settings.gradle

```diff
@@ -6,4 +6,5 @@ include 'TPM_Utils',
         'HIRS_Structs',
         'HIRS_AttestationCA',
         'HIRS_AttestationCAPortal',
+        'ACA_SwidTag',
```

Comment:
Do we need to add this sub-project to .travis.yml?

Reply:
Tested adding this subproject to Travis. Looks good to add it, if/when we need to.
https://travis-ci.org/nsacyber/HIRS/builds/632261783